tests/integration/components/task-card-test.js

```diff
@@ -174,8 +173,8 @@ test('assignment works if user has ability', function(assert) {
 
   renderPage();
 
-  page.taskAssignment.trigger.open();
-  page.taskAssignment.dropdown.options(0).select();
+  page.taskAssignment.select.trigger.open();
+  page.taskAssignment.select.dropdown.options(0).select();
```

Comment:
This one is on the right track. The other tasjs that call `page.render` directly should be replaced with `renderPage`, once `renderPage` is reverted back to the old version, where it renders the `task-card` component, not the `task-assignment` component